Tests/DispatchTests/DispatchQueueExtensionsTests.swift

```diff
@@ -51,6 +51,20 @@ final class DispatchQueueExtensionsTests: XCTestCase {
 
         waitForExpectations(timeout: 0.5, handler: nil)
     }
+    
+    func testAsyncAfter() {
+        let delay: Double = 2
+        var codeExecuted = false
+        let codeShouldBeExecuted = expectation(description: "Executed")
+        
+        DispatchQueue.main.asyncAfter(delay: delay) {
```

Comment:
I'm kinda surprised that this works, since I assumed that this is already running on, and therefore blocking, the main thread. It would make more sense to me for this to run on a background thread. But hey, if it works 🤷